sdk/src/main/java/com/eegeo/mapapi/indooroutlines/IndoorFloorOutlineInformation.java

```diff
@@ -0,0 +1,132 @@
+package com.eegeo.mapapi.indooroutlines;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.UiThread;
+import android.support.annotation.WorkerThread;
+
+import com.eegeo.mapapi.util.NativeApiObject;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ *  Maintains information about indoor map floor outline belonging to an indoor map floor with specified id.
```

Comment:
"about the indoor map floor outline or outlines"?

Reply:
Maintains information about one or more polygons representing the outline of an indoor map floor.

Reply:
Updated in b906189462f20f70457112ba3c85d183b7789d45